### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by ishan510**

### CODE REVIEW

File: static/photo-1534528741775-53994a69daeb.jpg
Comment (chrisvire) [inline static/photo-1534528741775-53994a69daeb.jpg:1]: Why is this here?

**Comment by chrisvire** [comment]
@ishan510 I saw a few relative path imports (e.g. ../../lib/fbconfig).  Please use "$lib/... (e.g. '$lib/fbconfig) instead.

File: static/photo-1534528741775-53994a69daeb.jpg
Comment (ishan510) [reply static/photo-1534528741775-53994a69daeb.jpg:1]: this is the avatar for the profile picture in the admin page

Comment (chrisvire) [reply static/photo-1534528741775-53994a69daeb.jpg:1]: Please do use this image but use Avatar placeholder with letters (based on email address or name if that is collected).

**Comment by chrisvire** [comment]
Feedback on the UI.
* I would float the avatar to the right end of the app bar.
* I would use the avatar placeholder (with letters) instead of the image.
* When the UI is narrow, it should have a hamburger icon (and not the "OPEN DRAWER" button to open the drawer.
* why does the background change when it gets narrow?
* Future: should have link to reset password
* Text of password should not be shown when typing.
* Optional: could have show/hide password icon but not needed for MVP

Minor:
* login text fields are not centered when the page is wide

**Review by chrisvire — APPROVED** [review]
Approved assuming the other issues will be addressed.
